.eslintrc

```diff
@@ -8,5 +8,7 @@ rules:
   max-len:
     - 2
     - 120
+  no-multi-assign: "off"
```

Comment:
I disabled these 2 rules because IMHO they don't offer us anything in this project